Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -1,74 +1,245 @@
 
-    ```
-     -----BEGIN CERTIFICATE-----
-     ...
-     -----END CERTIFICATE-----
-    ```
+<video controls>
+    <source src="https://github.com/demisto/content-assets/blob/master/Assets/MicrosoftTeams/FullConfigVideo.mp4?raw=true"
+            type="video/mp4"/>
+    Sorry, your browser doesn't support embedded videos. You can download the video at: https://github.com/demisto/content-assets/blob/master/Assets/MicrosoftTeams/FullConfigVideo.mov?raw=true
+</video>
 
-    ```
-     -----BEGIN PRIVATE KEY-----
-     ...
-     -----END PRIVATE KEY-----
-    ```
+## Prerequisites
 
-- You must not set a certificate and/or private key if you are using the Cortex XSOAR rerouting setup.
-- Microsoft does not support self-signed certificates and requires a chain-trusted certificate issued by a trusted CA.
- In order to verify which certificate is used, run the following (replace {MESSAGING-ENDPOINT} with the messaging endpoint):
+1. [Create the Demisto Bot in Microsoft Azure Portal](#1-create-the-demisto-bot-in-microsoft-teams)
+2. [Grant the Demisto Bot Permissions](#2-grant-the-demisto-bot-permissions-in-microsoft-graph)
+3. [Configure Microsoft Teams on Cortex XSOAR/XSIAM](#configure-microsoft-teams-on-cortex-xsoar/xsiam)
+4. [Add the Demisto Bot to a Team](#add-the-demisto-bot-to-a-team)
 
-    ```
-     curl {MESSAGING-ENDPOINT} -vI
-    ```
+### 1. Create the Demisto Bot in Microsoft Teams
 
-     Make sure the output does not contain the following:
+Creating the Demisto Bot using Microsoft Azure Portal:
 
-    ```
-     curl: (60) SSL certificate problem: self signed certificate
-    ```
+1. Navigate to the [Create an Azure Bot page](https://portal.azure.com/#create/Microsoft.AzureBot).
+2. In the Bot Handle field, type **Demisto Bot**.
+3. Fill in the required Subscription and Resource Group, relevant links: [Subscription](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription), [Resource Groups](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal).
+4. For Type of App, select **Multi Tenant**.
+5. For Creation type, select **Create new Microsoft App ID** for Creation Type if you don't already have an app registration, otherwise, select **Use existing app registration**, and fill in you App ID.
+    - **Note ⚠️:** if you choose **Use existing app registration**, make sure to delete the previous created bot with the same app id, remove it from the team it was added to as well.
+6. Click **Review + Create**, and wait for the validation to pass.
+7. Click **Create** if the validation has passed, and wait for the deployment to finish.
+8. Under Next Steps, click **Go to resource**.
+9. **Skip this step and return after configuring the Microsoft Teams instance in Cortex XSOAR/XSIAM**. Navigate to **Settings -> Configuration** on the left bar, and fill in the **Messaging Endpoint**.
```

Comment:
I am a bit lost as to where we are. Are we still under the Azure app? What happens after step 8 (Go to resources)? Not clear what I'm skipping.